Rubberduck.Resources/Rubberduck.Resources.csproj

```diff
@@ -12,6 +12,9 @@
     <Resource Include="**\*.bmp" Exclude="$(IntermediateOutputPath)\**" />
     <Resource Include="**\*.txt" Exclude="$(IntermediateOutputPath)\**" />
   </ItemGroup>
+  <ItemGroup>
+    <Resource Remove="Resources\light-bulb.png" />
+  </ItemGroup>
```

Comment:
huh? what's with that? Why is the file added, but excluded here?

Reply:
Grrr... I meant to remove that - I went with the existing one.  Will fix.